department-of-veterans-affairs/va.gov-cms
# VACMS-5544: Update vc help link.

## Description
closes #5544

## Testing done
Visual

## QA steps
Go to /node/add/vet_center and click on the `How to edit Vet Center pages.` link, visually verifying it takes you to the help page.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
